config/core/configmaps/leader-election.yaml

```diff
@@ -49,3 +49,10 @@ data:
     # retryPeriod is how long the leader election client waits between tries of
     # actions; 2 seconds is the value used by core kubernetes controllers.
     retryPeriod: "2s"
+
+    # buckets is the number of buckets used to partition key space of each
+    # Reconciler. If this number is M and the replica number of the controller
+    # is N, the N replicas will compete for the M buckets. The owner of a
+    # bucket will take care of the reconciling for the keys partitioned into
+    # that bucket.
```

Comment:
For StatefulSets, this must match the replica count.

Reply:
Which reminds me, do we still keep them, given we don't use them?

Reply:
I'm using it 😉

Reply:
Tell me more :)

Reply:
Currently we don't use StatefulSets and this flag is a global setting for all components. I think we should not mention StatefulSets at this point to avoid confusion.